Packs/NozomiNetworks/Integrations/NozomiGuardian/NozomiGuardian.py

```diff
@@ -0,0 +1,188 @@
+from CommonServerPython import *
+from CommonServerUserPython import *
+''' IMPORTS '''
+
+import urllib3
+
+# Disable insecure warnings
+urllib3.disable_warnings()
+
+
+''' CONSTANTS '''
+
+
+INTEGRATION_NAME = 'Nozomi Guardian'
+
+
+'''API Client'''
+
+
+class Client(BaseClient):
+
+    def query(self, search_query):
+        query = search_query
+        LOG('running request with url=%s' % self._base_url)
+
+        res = self._http_request(
+            "GET",
+            url_suffix='/api/open/query/do?query=' + query,
+            resp_type="json"
+        )
+        return res
+
+
+'''' Commands '''
+
+
+def test_module(client):
+    try:
+        client.query('help')
+    except Exception as e:
+        LOG(e)
+        return_error(f'Failed to execute {demisto.command()} command. Error: {str(e)}')
+    return 'ok'
+
+
+def search_by_query(client, args):
+    search_query = args.get('query')
+    try:
+        title = ("%s - Results for the Search Query" % INTEGRATION_NAME)
+        raws = []
+        nozomi_ec = []
+        raw_response = client.query(search_query)['result']
+
+        if raw_response:
+            raws.append(raw_response)
+            nozomi_ec.append({
+                'QueryResults': raws
+            })
+
+        if not raws:
+            return_error("%s - Could not find any results for given query" % INTEGRATION_NAME)
+
+        context_entry = {
+            "NozomiGuardian": {"Queries": nozomi_ec}
```

Comment:
why do you need so many layers? `NozomiGuardian.Queries.QueryResults`?
better to have: `NozomiGuardian.QueryResults`